src/utils/jsonable-value.ts

```diff
@@ -0,0 +1,35 @@
+import { stringify } from './json';
+
+/**
+ * Copy from `ts-jest`
+ * @internal
+ */
+export class JsonableValue<V = Record<string, unknown>> {
```

Comment:
Can we rename it to `JsonObject`?

This use case for this class is for caching, isn't it?

Reply:
yes, we can rename it. It is mainly for caching.